docs/concepts/troubleshooting-installed-packages.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Troubleshooting Installed Packages
+description: How to find which package source was used for individual packages
+author: JonDouglas
```

Comment:
You can put your name on the doc if you'd like updates on it / feedback notifications!